executor/executor_write.go

```diff
@@ -411,12 +417,18 @@ func (e *LoadDataInfo) InsertData(prevData, curData []byte) ([]byte, error) {
 		cols = escapeCols(rawCols)
 		e.insertData(cols)
 		e.insertVal.currRow++
+		if e.insertVal.batchRows != 0 && e.insertVal.currRow%e.insertVal.batchRows == 0 {
```

Comment:
I think use `mod` operator is not efficiently. You can let currRow = 0 every time it reaches batchRows, and then check currRow >= batchRows.